config/webpack/environment.js

```diff
@@ -1,3 +1,7 @@
 const { environment } = require('@rails/webpacker')
 
+environment.loaders.get('sass').use.splice(-1, 0, {
```

Comment:
**question:** what is this line doing?

Reply:
It is putting the `resolve-url-loader` into the proper place in the webpack config to actually resolve the fonts and images referenced in the USWDS sass files. Sourced from here: https://github.com/rails/webpacker/issues/2155#issuecomment-829741240

Reply:
Added a comment about that to the code

Reply:
non-blocking, but I don't see the comment! any chance you still need to push?

Reply:
It's there, but you can only see it in files view, for whatever reason. I think probably because I didn't change the exact line the initial comment was on?

Conversation tab:
<img width="811" alt="conversation_view" src="https://user-images.githubusercontent.com/285842/146014403-7decd1c7-f804-4b84-8ec9-6ef2984a872c.png">

Files tab:
<img width="1230" alt="files_view" src="https://user-images.githubusercontent.com/285842/146014443-ad70ff41-5da9-489b-9854-d9e59948f268.png">